### ISSUE TITLE
Info.plist does not exist in framework

### ISSUE THREAD

**Issue by mitchellallison** [PhilipsHue/PhilipsHueSDK-iOS-OSX#90]
On Xcode 6.3, I receive an error when trying to run code that links the Hue framework. The console log is as follows:
```
13/04/2015 13:30:50.613 com.apple.CoreSimulator.CoreSimulatorService[323]: Error Domain=LaunchServicesError Code=0 "The operation couldn’t be completed. (LaunchServicesError error 0.)" UserInfo=0x7fea6d67e9b0 {Error=PackageInspectionFailed, ErrorDescription=Failed to load Info.plist from bundle at path /Users/USER_NAME/Library/Developer/CoreSimulator/Devices/F704F4F9-C0F7-4AEE-84EF-F747F8D7DF59/data/Library/Caches/com.apple.mobile.installd.staging/temp.SyR7VI/extracted/HueControl.app/Frameworks/HueSDK.framework}
```

Manually adding an Info.plist to the framework seems to rectify the issue.

**Comment by lightbow** [comment]
Fixed now? There seems to be an Info.plist in the 1.8 beta SDK.

**Comment by jhvdb87** [comment]
This should be fixed since 1.8 beta.
